app.py

```diff
@@ -24,6 +20,24 @@ def __init__(self, *args, **kwargs):
         window.grid_rowconfigure(0, weight=1)
         window.grid_columnconfigure(0, weight=1)
 
+        # menu bar (wip) - maybe link functions here as well? -jh
```

Comment:
We can definitely consider tweaking these things to do things later on, but I like this item as a work-in-progress concept.